apps/fz_http/lib/fz_http_web/live/rule_live/rule_list_component.html.heex

```diff
@@ -40,6 +49,9 @@
                   <%= rule.destination %>
                 </dd>
               </td>
+              <td class="has-text-left">
+                <%= get_scoped_user(rule.user_id) %>
```

Comment:
This looks like an N+1? If we are already loading all users, we might as well build a map out of it and use it here.

Reply:
👍